docs/pages/database-access/guides/mysql-cloudsql.mdx

```diff
@@ -0,0 +1,254 @@
+---
+title: Database Access with MySQL on GCP Cloud SQL
+description: How to configure Teleport Database Access with GCP Cloud SQL MySQL.
+---
+
+# GCP Cloud SQL MySQL
+
```

Comment:
It would be nice to have a high level diagram to show people what the setup is needed.